specs/deepsea_su.json

```diff
@@ -5,7 +5,7 @@
     "env_size": [20, 40, 60, 80, 100],
     "n_episodes": 60000,
     "prior": 1,
-    "policy_arg": 1e-2,
+    "policy_arg": 1,
```

Comment:
Did you check whether this setting works well?

Reply:
It degrades the results, so tested with some values and seems 5.0 works as well as the original code.